lib/meta_tags/version.rb

```diff
@@ -1,4 +1,4 @@
 module MetaTags
   # Gem version.
-  VERSION = '2.1.0'
+  VERSION = '2.1.1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.